prime-router/build.gradle.kts

```diff
@@ -957,5 +933,8 @@ dependencies {
     testImplementation("org.testcontainers:junit-jupiter:1.19.6")
     testImplementation("org.testcontainers:postgresql:1.19.6")
 
+    testImplementation("io.ktor:ktor-server-test-host:$ktorVersion")
+    testImplementation("org.jetbrains.kotlin:kotlin-test:$kotlinVersion")
+    testImplementation("com.github.stefanbirkner:system-lambda:1.2.1")
```

Comment:
Why did we add these libraries?

Generally anything scoped to just tests is not a big deal but it adds more dependencies to keep up to date which we're specifically trying to reduce in this ticket.

Reply:
Nice catch! These were not necessary. I've reached out to Jim to find out why they were added but for now I've removed them.